src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,54 @@
 package core.basesyntax;
 
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
```

Comment:
Use LocalDate instead of Date for representing dates in Java, as it is part of the newer Java 8 Date-Time API which is more robust and flexible.